tests/roxctl/bats-tests/cluster/scanner-upload-db.bats

```diff
@@ -35,10 +35,10 @@ teardown() {
   assert_output --partial 'not a valid zip file'
 }
 
-@test "[zip] roxctl scanner upload-db" {
-  run curl --retry 30 --retry-max-time 600 --retry-all-errors --show-error --fail --output "${temp_dir}/test-scanner-vuln-updates.zip" --location 'https://install.stackrox.io/scanner/scanner-vuln-updates.zip'
-  assert_success
+# @test "[zip] roxctl scanner upload-db" {
```

Comment:
https://bats-core.readthedocs.io/en/stable/writing-tests.html#skip-easily-skip-tests
Please add a reason for this and todo to enable it back with issue with `ci_disabled` label